### PR TITLE
JobOffer seeds

### PR DESCRIPTION
**Pull request by emcoding**
Creating JobOffer seeds with FactoryGirl

### CODE REVIEW

**Comment by emcoding** [comment]
I think it's ready for pulling. Is it?

File: spec/factories/job_offers.rb [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
   factory :job_offer do
-    title         { FFaker::Lorem.sentence(5) }
-    description   { FFaker::Lorem.paragraph }
+    title         { FFaker::Job.title }
+    description   { FFaker::HipsterIpsum.paragraph }
     company_name  { FFaker::Company.name }
     contact_email { FFaker::Internet.email }
     location      { [ FFaker::Address.city, FFaker::Address.country ].join(', ') }
+
+    factory :job_offer_details do
```
Comment (carpodaster) [inline spec/factories/job_offers.rb]: Can you use a "trait" for it? For example `:with_details` ?

Comment (emcoding) [inline spec/factories/job_offers.rb]: Changed to :trait, it works, so yes, I can :-)

File: db/seeds.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-role = Role.create!([{name: 'organizer'}])
-user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
\ No newline at end of file
+# role = Role.create!([{name: 'organizer'}])
+# user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
+
+
+5.times do
+  FactoryGirl.create(:job_offer, :with_details)
```
Comment (carpodaster) [inline db/seeds.rb]: You can use `create_list` here instead of the loop

File: db/seeds.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 #
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-role = Role.create!([{name: 'organizer'}])
-user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
\ No newline at end of file
+# role = Role.create!([{name: 'organizer'}])
```
Comment (carpodaster) [inline db/seeds.rb]: Could you remove these commented lines?

And the unneeded newlines (here and in the very last line of the file)

**Comment by emcoding** [comment]
So, all my work of today is basically squashed into one line ;-) You know what Henry Ford said, when he sent a friend a 10 pages letter: "Sorry, I didn't have the time to write a short letter."

**Comment by carpodaster** [comment]
> I think it's ready for pulling. Is it?

@F3PiX it totally is!

**Comment by emcoding** [comment]
O @carpodaster, You made my week! I have to check some things, and I will ask one of the coaches to close it. THANK YOU (learned a lot from your reviews)

**Pull request merged**